extensions/xqc/package.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "xqc",
+  "title": "Xqc",
```

Shouldn't it be xQc?
```suggestion
  "title": "xQc",
```